CodeEdit/Features/AppPreferences/SoftwareUpdater.swift

```diff
@@ -30,6 +30,27 @@ class SoftwareUpdater: NSObject, ObservableObject, SPUUpdaterDelegate {
         }
     }
 
+    private var feedURLTask: Task<(), Never>?
+
+    private func setFeedURL() async {
+        let url = URL(string: "https://api.github.com/repos/CodeEditApp/CodeEdit/releases/latest")!
```

Comment:
Should we be setting these URL's in an Info.plist and referencing here instead of hardcoding?

Reply:
Let's for now keep it that way since this is just a quick fix.